src/themes/mat/colors.js

```diff
@@ -6,6 +6,8 @@ const colorsToShade= {
 	grey: '#7a7a7a',
 	darkGrey: '#4a4a4a',
 	type: '#333333',
+	typeGray: '#7a7a7a',
+	borderGray: '#9b9b9b',
```

Comment:
If you introduce variables in your theme, introduce them in the default theme as well.

Is there a way we can avoid these two new palette colors, though? For text, we already have type, typeLight and typeDisabled and white. For borders or shading, I would reach for white, grayTint, grayTintLight, grayTintLighter, darkness or darknessWhite.

Reply:
It occurs to me that in the rest of the project we have use grey with a British 'e', instead of gray with an American 'a'.

At the same time, other names are written with an American spelling. This did not cross my mind before I saw these variables next to each other here. Maybe we should use the american version across the board.